### PR TITLE
Terraform Templates for AWS

### PR DESCRIPTION
**Pull request by nickykln**
Please find the terraforms templates for an AWS env. 
Please let me know if some changes are required.

Hope it helps,

Nicolas

### CODE REVIEW

File: .DS_Store
Comment (Pascal-Delange) [inline .DS_Store:1]: to remove please :)

File: terraform_templates/.DS_Store
Comment (Pascal-Delange) [inline terraform_templates/.DS_Store:1]: ditto

**Review by Pascal-Delange — COMMENTED** [review]
Thank you ! 
Since I'm not particularly proficient with AWS infra, I won't review the details of the manifest files too deeply, though I had a look at them.
May I just ask you to give a short description of how you are deploying Marble with this setup ? Is it all managed containers, similarly to what we are doing on GCP with Cloud Run ?
And let's maybe add (for clarity) a last update date in the readme (we won't be able to keep those files updated with future changes in the app, this way at least it's clear how fresh they are)

**Review by balzdur — APPROVED** [review]

**Comment by Pascal-Delange** [comment]
Thanks !

**Comment by Pascal-Delange** [comment]
I'll fix conflicts and rebase later this week

**Comment by Pascal-Delange** [comment]
Hello @nickykln ,
I created a branch in my base repo to fix merge conflicts from your branch, which I merged in https://github.com/checkmarble/marble/pull/39
Hence, I'll close your PR because its commits have been backported.
Not sure if that was the most elegant solution, sorry if it's not the case.

**Pull request closed without merging**